Gsk-4.0.gir

```diff
@@ -4221,33 +4210,6 @@ with the given @transform.</doc>
         </parameters>
       </method>
     </class>
-    <function-macro name="VULKAN_RENDERER" c:identifier="GSK_VULKAN_RENDERER" introspectable="0">
-      <parameters>
-        <parameter name="obj">
-        </parameter>
-      </parameters>
-    </function-macro>
-    <function-macro name="VULKAN_RENDERER_CLASS" c:identifier="GSK_VULKAN_RENDERER_CLASS" introspectable="0">
-      <parameters>
-        <parameter name="klass">
-        </parameter>
-      </parameters>
-    </function-macro>
-    <function-macro name="VULKAN_RENDERER_GET_CLASS" c:identifier="GSK_VULKAN_RENDERER_GET_CLASS" introspectable="0">
-      <parameters>
-        <parameter name="obj">
-        </parameter>
-      </parameters>
-    </function-macro>
-    <class name="VulkanRenderer" c:symbol-prefix="vulkan_renderer" c:type="GskVulkanRenderer" parent="Renderer" glib:type-name="GskVulkanRenderer" glib:get-type="gsk_vulkan_renderer_get_type" glib:type-struct="VulkanRendererClass">
-      <doc xml:space="preserve">A GSK renderer that is using Vulkan.</doc>
-      <constructor name="new" c:identifier="gsk_vulkan_renderer_new">
-        <return-value transfer-ownership="full">
-          <type name="Renderer" c:type="GskRenderer*"/>
-        </return-value>
-      </constructor>
-    </class>
-    <record name="VulkanRendererClass" c:type="GskVulkanRendererClass" disguised="1" glib:is-gtype-struct-for="VulkanRenderer"/>
```

Comment:
I suppose this is caused by gtk no longer building with vulkan by default. I wonder if it's fine to not have it as part of the API or still build gtk with vulkan enabled somehow?

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/commit/4d5dc18a575ce76b5f2469832c8d0d805b355bd9

Or we can ask the gnome SDK to enable it? But that commit message looks like it's not yet ready for general use.

Reply:
So I asked the GTK developers and mclassen said 
> "default-off experimental api doesn't need bindings" would be an acceptable stance, maybe

Which sounds good to me. I don't expect the GNOME sdk to have vulkan enabled if it's causing issues for everyone even if they are not using it.